client/src/__tests__/RelatedItems.test.js

```diff
@@ -191,3 +209,87 @@ describe('Carousel component', () => {
     expect(axios.get).toHaveBeenCalledTimes(10);
   });
 });
+
+describe('ComparisonModal Component', () => {
+  const productInfo = {
+    name: 'Heir Force Ones',
+    category: 'Kicks',
+    features: [
+      {
+        feature: 'Mid-Sole',
+        value: 'ControlSupport Arch Bridge',
+      },
+      {
+        feature: 'Material',
+        value: 'Double Stitch',
+      },
+    ],
+  };
+  const mockAxiosVal = {
+    data: {
+      name: 'Morning Joggers',
+      category: 'Pants',
+      features: [
+        {
+          feature: 'Sole',
+          value: 'true',
+        },
+        {
+          feature: 'Material',
+          value: 'FullControlSkin',
+        },
+      ],
+    },
+  };
+  const setShowModal = jest.fn();
+
+  test('should render the compared product name and the current product name', async () => {
+    axios.get.mockImplementation(() => Promise.resolve(mockAxiosVal));
```

Comment:
consider adding this line in a beforeEach() statement outside of your tests so you don't have to repeat it so often.